### PR TITLE
[15.0][FW]base_user_role: Groups-roles navigation

### PR DESCRIPTION
**Pull request by GuillemCForgeFlow**
@ForgeFlow
ForwardPort of https://github.com/OCA/server-backend/pull/157 from 13.0 to 15.0.

### CODE REVIEW

File: base_user_role/views/group.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" ?>
```
Comment (MiquelRForgeFlow) [inline base_user_role/views/group.xml]:
```suggestion
<?xml version="1.0" encoding="utf-8"?>
```

Comment (GuillemCForgeFlow) [reply base_user_role/views/group.xml]: Thanks!! :smile:

**Review by HaraldPanten — APPROVED** [review]
Functional review

**Review by MiquelRForgeFlow — APPROVED** [review]

**Comment by sebalix** [comment]
Same remark than in https://github.com/OCA/server-backend/pull/159 to preserve the original commit

**Review by sebalix — COMMENTED** [review]
Same remark than in https://github.com/OCA/server-backend/pull/159 to preserve the original commit

**Review by sebalix — APPROVED** [review]

**Comment by sebalix** [comment]
@GuillemCForgeFlow Runboat isn't happy, can you check?

**Comment by MiquelRForgeFlow** [comment]
Runboat isn't happy because he reintroduced by error same previous bugs:
Selection_340

**Comment by GuillemCForgeFlow** [comment]
> Runboat isn't happy because he reintroduced by error same previous bugs: Selection_340

Fixed! :wink:

**Review by sebalix — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Comment by sebalix** [comment]
@GuillemCForgeFlow thanks for the follow-up on all versions!

/ocabot merge minor

File: base_user_role/models/__init__.py [hunk 2]
```diff
@@ -1,3 +1,3 @@
 from . import role
 from . import user
-from . import res_groups
+from . import group
```
Comment (StefanRijnhart) [inline base_user_role/models/__init__.py]: By renaming the file to what it was in Odoo 13, you are moving *away* from the OCA convention. In this particular case, it also makes it more difficult to check what has changed in this file. Would you consider keeping the original name?

Comment (GuillemCForgeFlow) [reply base_user_role/models/__init__.py]: You are right, I've applied the changes you are commenting.

Comment (MiquelRForgeFlow) [reply base_user_role/models/__init__.py]: You have to change the name of the file also :S

Comment (GuillemCForgeFlow) [reply base_user_role/models/__init__.py]: @MiquelRForgeFlow Thanks :smile:

**Review by StefanRijnhart — APPROVED** [review]

**Pull request merged**